src/pages/components/Card/Card.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Tooltip } from '@chakra-ui/react';
+import { ReactI18NextChild } from 'react-i18next';
+
+interface CardPropsInterface {
+  title: string;
+  description: string;
+  icon: ReactI18NextChild;
```

Comment:
Hi @ijemmao, with your suggestion here, I got:  `Property 'icon' does not exist on type 'ButtonProps'.` error.

Reply:
`React.ReactNode` could also be used here but I am not sure that best describes an icon.

Reply:
it would be `ButtonProps['rightIcon']` or `ButtonProps['leftIcon']` - the main focus being that you want to extend from the predefined `ButtonProps` type so you don't have to define these types by hand